analyzers/integration_tests/analyzers/nodejs_test.go

```diff
@@ -0,0 +1,79 @@
+package nodejsanalyzer
+
+import (
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/fossas/fossa-cli/analyzers"
+	"github.com/fossas/fossa-cli/analyzers/integration_tests/project_initializer/node"
+	"github.com/fossas/fossa-cli/analyzers/integration_tests/testtools"
+	"github.com/fossas/fossa-cli/files"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+	"github.com/stretchr/testify/assert"
+)
+
+var nodeInitializer = node.New()
+
+func TestMain(m *testing.M) {
```

Comment:
This should be part of the Node.JS analyzer tests. In general, tests should be colocated with the package being tested.